src/AppInstallerCLICore/Workflows/PromptFlow.cpp

```diff
@@ -164,7 +164,7 @@ namespace AppInstaller::CLI::Workflow
                     }
                 }
 
-                EnsurePackageAgreementsAcceptance(context, /* showPrompt */ false);
+                EnsurePackageAgreementsAcceptance(context, /* showPrompt */ true);
```

Comment:
Nit: If we do this, the `showPrompt` argument isn't needed anymore and could be removed

Reply:
Discussed offline, we'll leave this as is in case we found issues and need to revert back